File: src/main/java/ejercicio10/Main.java
package ejercicio10;

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner lectura = new Scanner (System.in);
        int eleccion = impMenu(lectura);
        System.out.println("Ahora dime el numero que quieres transformar: ");
        Float num = lectura.nextFloat();

        if (eleccion == 1) {
            System.out.println( num * 9/5 + 32);
        }else if (eleccion == 2){
            System.out.println((num - 32) * 5/9);
        }else System.out.println("Numero introducido no valido");

    }

    public static int impMenu(Scanner scanner){
        System.out.println("Que combersion quieres hacer?: ");
        System.out.println("1. Celsius a Fahrenheit ");
        System.out.println("2. Fahrenheit a Celsius ");
        return scanner.nextInt();
    }
}

File: src/main/java/ejercicio10/Main.kt
package ejercicio10

fun main() {
    val eleccion = impMenu()
    println("Ahora dime el numero que quieres transformar: ")
    val num = readln().toFloat()

    when (eleccion) {
        1 -> println(num * 9 / 5 + 32)
        2 -> println((num - 32) * 5 / 9)
        else -> println("Numero introducido no valido")
    }
}

fun impMenu(): Int {
    println("¿Qué conversión quieres hacer?: ")
    println("1. Celsius a Fahrenheit")
    println("2. Fahrenheit a Celsius")
    return readln().toInt()
}
